src/Tribe/Assets.php

```diff
@@ -327,19 +327,24 @@ public static function maybe_get_min_file( $url ) {
 		// Strip the plugin URL and make this relative.
 		$relative_location = str_replace( $base_url, '', $url );
 
+		if ( defined( 'SCRIPT_DEBUG' ) && true === SCRIPT_DEBUG ) {
+			// Add the actual url after having the min file added.
+			$urls[] = $relative_location;
+		}
+
 		// If needed add the Min Files.
-		if ( ! defined( 'SCRIPT_DEBUG' ) || SCRIPT_DEBUG === false ) {
-			if ( substr( $relative_location, - 3, 3 ) === '.js' ) {
-				$urls[] = substr_replace( $relative_location, '.min', - 3, 0 );
-			}
+		if ( substr( $relative_location, - 3, 3 ) === '.js' ) {
```

Comment:
Space on unary operator. 
```suggestion
		if ( substr( $relative_location, -3, 3 ) === '.js' ) {
```